Graylog2/graylog2-server
# Rule var renumber

/jpd Graylog2/graylog2-server#16233
/nocl new feature
Relates to #16303 

Normalize rule when creating or updating (in particular: renumber generated output variables to start from 1).

**To test:** 
- use rule builder to create a rule with multiple output variables (e.g. from capitalize function)
- delete some of the actions
- click update
- note that output variables are renumbered
